cfe_internal/enterprise/federation/federation.cf

```diff
@@ -292,8 +292,10 @@ bundle agent federation_manage_files
         copy_from => default:local_dcp( "$(this.promise_dirname)/../../../templates/federated_reporting/parallel.sh" ),
         perms => default:mog( "640", "root", "$(transport_user)" );
 
-      "$(cfengine_enterprise_federation:config.bin_dir)/psql_wrapper.sh"
-        copy_from => default:local_dcp( "$(this.promise_dirname)/../../../templates/federated_reporting/psql_wrapper.sh" ),
+      "$(cfengine_enterprise_federation:config.bin_dir)/psql_wrapper.sh" -> { "ENT-4792"}
```

Comment:
@nickanderson Wrong promisee ticket?

Reply:
No, thats just a bread crumb for why i had to do create => true for future cleanup